example_zoo/tools/tf_models_samples.yaml

```diff
@@ -1,5 +1,6 @@
 org: tensorflow
 repository: models
+runtime_version: "1.13"
```

Comment:
nit: We are in 1.14 now

Reply:
this needs to wait until tf/models release their 1.14.0 examples, which should be soon.